src/api/index.ts

```diff
@@ -17,16 +17,16 @@ export const getTopReposAsync = async (
   page: number,
   sort: string,
   language?: string,
-  topic?: string
+  topics?: string[]
 ): Promise<Repos> => {
   let q = `${sort}:>${MIN_COUNT}`;
 
   if (language !== undefined && language.trim() !== '') {
     q += ` language:"${language}"`;
   }
 
-  if (topic !== undefined && topic.trim() !== '') {
-    q += ` topic:${topic}`;
+  if((topics != null) && (topics.length > 0)) {
+    q += topics.map(topic => ` topic:${topic}`).join('+');
```

Comment:
For multiple topics, the query string should be `topic:react topic:javascript` without `+`.